### PR TITLE
[types/chrome] Fix wrong type of manifest author key: string -> {email: string}

### PR DESCRIPTION
**Pull request by James-ZHANG**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test chrome`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others: changing an existing definition.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://developer.chrome.com/docs/extensions/reference/manifest/author
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header. **Does not apply: Chrome extension API implementation is Chrome-native.**

### CODE REVIEW

File: types/chrome/index.d.ts [hunk 1]
```diff
@@ -7235,7 +7235,9 @@ declare namespace chrome.runtime {
         icons?: ManifestIcons | undefined;
 
         // Optional
-        author?: string | undefined;
+        author?: {
+            email: string;
+        } | undefined;
```
Comment (matcarlson) [inline types/chrome/index.d.ts:7240]: Can you add a comment like this, from https://developer.chrome.com/docs/extensions/reference/manifest/author? 

```suggestion
        author?: {
            // This is the email address of the extension author. When 
            // publishing a CRX file to the Chrome Web Store, this string MUST
            // match the email address of the account used to publish the
            // extension.
            email: string;
        } | undefined;
```

Comment (nicolas377) [reply types/chrome/index.d.ts:7240]: I think the precedent with manifest is to leave out the comments. We're not setting the standard for the actual .json file, we can't get that intellisense yet. I don't see a point in adding a comment here when authors are already  looking at chrome docs for the manifest.

Comment (matcarlson) [reply types/chrome/index.d.ts:7240]: That's fair, but there are folks who generate their JSON file through TypeScript (I do this weirdly enough). I'm OK with either but still generally prefer to have some documentation.

**Review by matcarlson — APPROVED** [review]
Approved - Feel free to add the comments I mentioned if you want and I'll re-approve. Thanks for your contribution.

**Comment by James-ZHANG** [comment]
Ready to merge

**Pull request merged**